beacon-chain/rpc/beaconv1/state.go

```diff
@@ -174,12 +174,18 @@ func (bs *Server) state(ctx context.Context, stateId []byte) (*statetrie.BeaconS
 		}
 	case "finalized":
 		checkpoint := bs.ChainInfoFetcher.FinalizedCheckpt()
+		if checkpoint == nil {
```

Comment:
https://github.com/prysmaticlabs/prysm/blob/develop/beacon-chain/blockchain/chain_info.go#L72

It isn't possible to return a nil checkpoint here.